.github/labeler.yml

```
@@ -35,3 +35,8 @@
 
 'pkg: vue':
 - packages/integrations/vue/**
+
+'documentation pr':
```

Just a suggestion, not blocking. Is just "docs" faster to scan?
```suggestion
'docs pr':
```